src/reducers/MessageCenter.ts

```diff
@@ -0,0 +1,101 @@
+import { MessageCenterActionType } from '../actions';
+import { MessageType } from '../types/MessageCenter';
+import { MessageCenterState } from '../store/Store';
+
+const INITIAL_STATE: MessageCenterState = {
+  nextId: 0,
+  groups: [
+    {
+      id: 'default',
+      title: 'Default',
+      messages: []
+    }
+  ],
+  hidden: true,
+  expanded: false,
+  expandedGroupId: 'default'
+};
+
+const mergeToState = (prevState, toMerge) => {
+  return Object.assign({}, prevState, toMerge);
```

Comment:
can this be shortened to just the following?

```const mergeToState = (prevState, toMerge) => ( { ...prevState, toMerge } )```

Reply:
Yes! to `const mergeToState = (prevState, toMerge) => ( { ...prevState, ...toMerge } )`